functests/5_latency_testing/latency_testing.go

```diff
@@ -286,7 +286,7 @@ func getValidValuesTests(toolToTest string) []latencyTest {
 	if toolToTest == hwlatdetect {
 		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: successRuntime, testMaxLatency: "1", hwlatdetectMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
 		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: successRuntime, hwlatdetectMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
-		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: successRuntime, outputMsgs: []string{success}, toolToTest: toolToTest})
+		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: successRuntime, testMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
```

Comment:
the idea is fine, but the variable name suggests this is the expected latency for guaranteed workloads rather than an arbitrary huge value just to exercise the tool

Reply:
Yep and the value of it is also pretty low.

Reply:
sure, we can update that. Is safeLatencyThreshold sounds better? 
I'll add a comment to clarify this constant usage and increase the threshold. If you have any preferred threshold in mind that is enough to be counted as safe you are welcome to share.

Reply:
what about `UntunedLatencyThreshold` or something like it?
Unless the only usage of `guaranteedLatency` is right here, I recommend adding (not replacing) a new constant + adding the comment you suggested to it

Reply:
sounds better, thanks!
Yes, it's only used in these tests for the same reason (not to fail on high latency results)

Reply:
> Yep and the value of it is also pretty low.

@MarSik the value was updated to 900000 in NTO. will update it here too.

Reply:
actually this was addressed in another PR (I assume it wasn't merged once you reviewed this one):
https://github.com/openshift-kni/performance-addon-operators/pull/964